transports/azure-service-bus/operational-scripting_migration-endpoint-command_asbs_[5,).partial.md

```diff
@@ -0,0 +1,120 @@
+## Migration-specific commands
+
+These commands are dedicated to endpoints that are configured with the migration topology for the purpose of migrating from the single-topic publishing approach (forwarding topology) to topic-per-event approach.
+
+- `migration endpoint create`
+- `migration endpoint subscribe`
+- `migration endpoint unsubscribe`
+- `migration endpoint subscribe migrated`
+- `migration endpoint unsubscribe migrated`
+
+### asb-transport migration endpoint create
+
+Creates infrastructure for an endpoint: input queue, topic, and subscription.
+
+```
+asb-transport migration endpoint create name
+                              [--size]
+                              [--partitioned]
+                              [--topic]
+                              [--topic-to-publish-to] [--topic-to-subscribe-on]
+                              [--subscription]
+```
+
+#### options
+
+`-c` | `--connection-string` : Overrides the environment variable 'AzureServiceBus_ConnectionString'
+
+`-n` | `--namespace` : Sets the fully qualified namespace to connect with cached credentials, e.g., credentials from Azure PowerShell or CLI. This setting cannot be used in conjunction with the connection string setting.
+
+`-s` | `--size` : Queue size in GB (defaults to 5)
+
+`-p` | `--partitioned`: Enable partitioning
+
+`-t` | `--topic`: Topic name (defaults to 'bundle-1')
+
+`-tp` | `--topic-to-publish-to`: The topic name to publish to.
+
+`-ts` | `--topic-to-subscribe-on`: The topic name to subscribe on.
+
+`-b` | `--subscription`: Subscription name (defaults to endpoint name)
+
+### asb-transport migration endpoint subscribe
+
+Creates a new subscription for an endpoint using single-topic approach.
+
+```
+asb-transport migration endpoint subscribe name event-type
+                              [--topic]
+                              [--subscription]
+                              [--rule-name]
+```
+
+#### Options
+
+`-c` | `--connection-string` : Overrides the environment variable 'AzureServiceBus_ConnectionString'
+
+`-n` | `--namespace` : Sets the fully qualified namespace to connect with cached credentials, e.g., credentials from Azure PowerShell or CLI. This setting cannot be used in conjunction with the connection string setting.
+
+`-t` | `--topic`: Topic name to subscribe on (defaults to 'bundle-1')
+
+`-b` | `--subscription`: Subscription name (defaults to endpoint name)
+
+`-r` | `--rule-name`: Rule name (defaults to event type)
```

Comment:
Aren't we having a similar problem here to what we discussed on the tool PR. I think "defaults to" is ambiguous here. As I would read, it is that both the subscription name and rule name are optional... This comment also applies for the other "defaults to"

Reply:
The rest looks good to me